### PR TITLE
Add homepage and basic components

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          type="text"
+        />
+      </div>
+
+      <div className="nav-innerContainer nav-icon-container width-reset inherit-clr mr-1">
+        <div className="flex-col-center">
+          <a className="font-clr" href="#">
+            <i className="fa-solid fa-heart"></i>
+          </a>
+          <small>Wishlist</small>
```
Comment (mikesch803) [inline src/components/Navbar/Navbar.jsx:27]: i guess you dont wishlist in note app

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <i className="fa-solid fa-user"></i>
+          </a>
+          <small>Profile</small>
+        </div>
+
+        <div className="flex-col-center">
+          <a className="font-clr nav-icon" href="#">
+            <i className="fa-solid fa-cart-shopping"></i>
+          </a>
+          <small>Cart</small>
```
Comment (mikesch803) [inline src/components/Navbar/Navbar.jsx:41]: here you also add cart

File: src/components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <li className="videolib-list-item sidebar-list">
+          <i className="fa-solid fa-box-archive videolib-drawer-icon"></i> Archive
+        </li>
+        <li className="videolib-list-item sidebar-list">
+          <i className="fa-solid fa-trash-can videolib-drawer-icon"></i> Trash
+        </li>
+        <li className="videolib-list-item sidebar-list">
+          <i className="fa-solid fa-pencil videolib-drawer-icon"></i>
+          Edit labels
+        </li>
```
Comment (mikesch803) [inline src/components/Sidebar/Sidebar.jsx:23]: suggestion : here is repeated code, so you can make array, map here

**Review by mikesch803 — APPROVED** [review]

File: src/pages/Notes/Notes.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            <div>
+              <small className="font-size-small ml-1 f-weight-500">
+                PINNED
+              </small>
+              <div className="d-flex notecard-container">
+                <NoteCard />
+              </div>
+            </div>
+            <div>
+              <small className="font-size-small ml-1 f-weight-500">
```
Comment (hsnice16) [inline src/pages/Notes/Notes.jsx:31]: Here we can improve the code a little bit, not much. 
As the element structure for the `PINNED` and `OTHERS` are same almost, so you can loop over an array.
Something like,
```
[{_id: 001, value: "PINNED"}, {...other}].map(({_id, value}) => (
<div>
    <small>
       ... all others tag
))
```

See, if you found this interesting, you can refactor your code.

File: src/pages/Auth/Login/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  const [userLoginData, setUserLoginData] = useState({
+    email: "",
+    password: "",
+  });
+  const [passwordInputType, setPasswordInputType] = useState("password");
+
+  const navigate = useNavigate();
+
+  function handleUserLogin() {
+    (async () => {
```
Comment (hsnice16) [inline src/pages/Auth/Login/Login.jsx:16]: Nice, that you are using IIFE here.

**Review by hsnice16 — APPROVED** [review]
Nice work, the code quality is good. Have added a few suggestions.
Good Job.

File: src/pages/Notes/Notes.jsx (hunk 4 above)
Comment (abhisheksahani-nova) [reply src/pages/Notes/Notes.jsx:31]: Thanks I will look into it

File: src/components/Sidebar/Sidebar.jsx (hunk 3 above)
Comment (abhisheksahani-nova) [reply src/components/Sidebar/Sidebar.jsx:23]: Thanks I will look into it

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/components/Navbar/Navbar.jsx:27]: yes

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (abhisheksahani-nova) [reply src/components/Navbar/Navbar.jsx:41]: yaah I forgot that

**Pull request merged**